examples/clients.js

```diff
@@ -0,0 +1,25 @@
+const low = require('lowdb');
+const FileSync = require('lowdb/adapters/FileSync');
+
+const adapter = new FileSync('clientsDb.json');
+const db = low(adapter);
+
+db.defaults({posts: [], user: {}, count: 0}).write();
+
+// Get registered client by issuer
+const get = async (iss) => {
+  const client = await db.get({iss});
+  if (client) return client;
+
+  console.log(
+    'No client found.  Returning null to initiate dynamic registration'
```

Comment:
Is it the eslint formatting? I'd assume it to be formatted as:

```
console.log(
     'No client found.  Returning null to initiate dynamic registration');
```

No worries if that's the auto formatting, let's keep it this way if so.

Reply:
No, eslint mainly checks the formatting, can do some autoformat on `--fix`, but this is my writing. `eslint` didn't complain though, I wonder if there is a rule to enforse ending parenthesis on the same line even if initial parenthesis ended its own line...  I'll look into it.

Do you want 4 spaces indentation in js?  I could add .editorconfig as well...

Reply:
We use 2 space indents for blocks (wrapped in `{}`) like function, object declaration, if/then/switch/while blocks etc.
We use 4 spaces indents for wrapping new lines or function arguments that don't fit on one line.

It's great if eslint can fix formatting. We could add it as a requirement before a git commit (it would check the formatting and if not good, ask to fix it with `eslint`)